src/mammos_mumag/materials.py

```diff
@@ -108,6 +108,11 @@ def add_domain(
             theta: Angle of the magnetocrystalline anisotropy axis
                 from the :math:`z`-direction in radians.
 
+        Examples:
+            >>> from mammos_mumag.materials import Materials
+            >>> mat = Materials()
+            >>> mat.add_domain(A=1, Ms=2, K1=3, K2=0, phi=0, theta=0)
```

Comment:
```suggestion
            >>> mat.add_domain(A=1, Ms=2, K1=3, K2=0, phi=0, theta=0)
            >>> mat
            TODO
```
Please add a line to show `mat` to (i) document how it looks and (ii) have something to test.